docs/nodes/AI_ML/CLASSIFICATION/ACCURACY/examples/EX1/example.md

```diff
@@ -1,5 +1,12 @@
-In this example, the [iris dataset](https://archive.ics.uci.edu/dataset/53/iris) is split into two parts, one for training and the other for testing. The labels from the test data are stripped using an `EXTRACT_COLUMNS` node, taking only the features of the data. 
+---
+title: Running an Accuracy Classification Model in Python with Flojoy
+description: Example of an Accuracy Classification Model in Python with Flojoy
```

Comment:
Perhaps start with "How to...".

Also, what are the SEO guidelines for the length of the description? Is there a minimum advised length? Is this perhaps a little short?